api/halls/halls.go

```diff
@@ -0,0 +1,63 @@
+package halls
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	halldb "github.com/darkjedidj/cinema-service/internal/repository/halls"
+	"github.com/gorilla/mux"
+)
+
+var Repo halldb.Repository
+
+//Handle handles all endpoints on this route
+func Handle(response http.ResponseWriter, request *http.Request) {
+	vars := mux.Vars(request)
+	route := vars["route"]
+	switch route {
+	case "post":
```

Comment:
WTF is this?

Reply:
couldnt understand what do u mean? naming? switch case usage? way i get parameters?

Reply:
Describe the endpoints which you are going to handle here